src/ex2/PokemonClient.js

```diff
@@ -0,0 +1,32 @@
+class PokemonClient {
+    constructor() {
+        this.API_BASE = "https://pokeapi.co/api/v2/pokemon-form/";
```

Comment:
no need to put this on the this, you can just set it as a const at top of file and then the casing makes sense as well :)